filename

```diff
@@ -0,0 +1 @@
+tickets/template_default.html
```

Comment:
What's this file?

Reply:
I'm not sure i will ask to francisco he create this file for the hackaton. That is one of muy questions when i did this part.